js/global.js

```diff
@@ -774,21 +778,32 @@ jQuery(document).ready(function($) {
 
     // Activate ToggleFlyout and ButtonGroup menus
     $(document).delegate('.ButtonGroup > .Handle', 'click', function() {
-        var buttonGroup = $(this).closest('.ButtonGroup');
-        if (buttonGroup.hasClass('Open')) {
+        var $buttonGroup = $(this).closest('.ButtonGroup');
+        if ($buttonGroup.hasClass('Open')) {
             // Close
-            $('.ButtonGroup').removeClass('Open');
+            $('.ButtonGroup').each(function(){
+                $(this).removeClass('Open');
+                $(this).setFlyoutAttributes();
```

Comment:
Shouldn't both these calls work on all matched elements and thus not need the `each()`?